### PR TITLE
SSCS-10178 Filter By Venue

### PR DESCRIPTION
**Pull request by TeaWithLucas**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SSCS-10178

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/service/hmc/topic/HearingUpdateService.java [hunk 1]
```diff
@@ -97,4 +102,30 @@ public void setHearingStatus(String hearingId, @Valid SscsCaseData sscsCaseData,
...
+
+    public void setWorkBasketFields(String hearingId, @Valid SscsCaseData sscsCaseData, HmcStatus hmcStatus) {
+
+        WorkBasketFields workBasketFields = sscsCaseData.getWorkBasketFields();
+
+        if (isCaseListed(hmcStatus)) {
+            String hearingEpimsId = getHearingEpimsId(hearingId, sscsCaseData);
+            workBasketFields.setHearingEpimsId(hearingEpimsId);
+        } else {
+            workBasketFields.setHearingEpimsId(null);
```
Comment (MohammedHMCTS) [inline src/main/java/uk/gov/hmcts/reform/sscs/service/hmc/topic/HearingUpdateService.java:114]: Maybe an early return on this if(!isCaseListed(hmcStatus){ return; }

**Review by MohammedHMCTS — DISMISSED** [review]
LGTM, One minor comment not required though

File: src/main/java/uk/gov/hmcts/reform/sscs/service/hmc/topic/HearingUpdateService.java (hunk 1 above)
Comment (TeaWithLucas) [reply src/main/java/uk/gov/hmcts/reform/sscs/service/hmc/topic/HearingUpdateService.java:114]: It needs to be set to null in that case, not return.

Comment (MohammedHMCTS) [reply src/main/java/uk/gov/hmcts/reform/sscs/service/hmc/topic/HearingUpdateService.java:114]: Sorry  if(!isCaseListed(hmcStatus){ workBasketFields.setHearingEpimsId(null); return; } <- I meant this

Comment (TeaWithLucas) [reply src/main/java/uk/gov/hmcts/reform/sscs/service/hmc/topic/HearingUpdateService.java:114]: I'd perfer not to do that in this case, it doesn't give any benefits and I think this is clearer.

**Review by dattipoeHMCTS — APPROVED** [review]

**Review by wmarshid — APPROVED** [review]

**Review by dattipoeHMCTS — APPROVED** [review]

**Review by wmarshid — APPROVED** [review]

**Pull request merged**